server/devtool/src/main/java/org/eclipse/vorto/server/devtool/service/IInformationModelEditorService.java

```diff
@@ -0,0 +1,21 @@
+package org.eclipse.vorto.server.devtool.service;
+
+import java.util.List;
+import java.util.Set;
+
+import org.eclipse.emf.ecore.resource.ResourceSet;
+import org.eclipse.vorto.http.model.ModelId;
+import org.eclipse.vorto.http.model.ModelResource;
+
+/**
+ * 
+ * @author shiv
+ *
+ */
+public interface IInformationModelEditorService {
+
+	String linkFunctionBlockToInformationModel(String infoModelResourceId, ModelId functionBlockModelId, ResourceSet resourceSet,
+			Set<String> referencedResourceSet);
+
+	List<ModelResource> searchFunctionBlockByExpression(String expression);
```

Comment:
My reason for making a REST call from the backend was so that I could perform a type check on the search results that are being returned from the Vorto repository. 
Secondly I kept this so that in case we need to add some further processing for the results later on, that can be done at the backend itself.

However, if this type check is not required, I'll move this call to the front end then